main/zoe/api/zoe-helpers.md

```diff
@@ -147,6 +147,27 @@ canTradeWith(leftInvite, rightInvite)
 canTradeWith(leftInvite, cantTradeRightInvite)
 ```
 
+## zoeHelpers.canTradeWithMapKeywords(leftOfferHandle, rightOfferHandle, keywords)
+- `leftOfferHandle`
+- `rightOfferHandle`
+- `keywords` - correspondence map of keywords on the left and right handles. keywords is an
+  array containing two equal-length arrays.
+- Returns: `{Boolean}`
+
+Checks if the `give` and `want` of two invites would satisfy offer
```

Comment:
This description of the handle arguments should also be up in the spec on lines 151-2.